ra_aid/llm.py

```
@@ -1,93 +1,184 @@
+    if is_expert:
+        temperature = 0
+
+    if provider == "deepseek":
+        return create_deepseek_client(
+            model_name=model_name,
+            api_key=config["api_key"],
+            base_url=config["base_url"],
+            temperature=temperature,
+            is_expert=is_expert,
+        )
+    elif provider == "openrouter":
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)